packages/react/src/Button/Button.tsx

```diff
@@ -6,26 +6,31 @@
 import clsx from 'clsx'
 import { forwardRef } from 'react'
 import type { ButtonHTMLAttributes, ForwardedRef, PropsWithChildren } from 'react'
+import { Icon } from '../Icon'
 
 export type ButtonProps = {
   /** The level of prominence. Use a primary button only once per page or section. */
   variant?: 'primary' | 'secondary' | 'tertiary'
+  icon?: Function
+  iconPosition?: 'start' | 'only'
```

Comment:
Reordered and added prop descriptions:

```suggestion
  /** An icon to add to the button. */
  icon?: Function
  /** The position of the icon. The default is after the label. */
  iconPosition?: 'start' | 'only'
  /** The level of prominence. Use a primary button only once per page or section. */
  variant?: 'primary' | 'secondary' | 'tertiary'
```